daffodil-schematron/src/test/resources/xsd/embedded-1.dfdl.xsd

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+           elementFormDefault="qualified">
+
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+    <xs:annotation>
+        <xs:appinfo source="http://www.ogf.org/dfdl/">
+            <dfdl:format ref="GeneralFormat" />
+        </xs:appinfo>
+    </xs:annotation>
+
+    <xs:element name="uuid" type="xs:string" dfdl:lengthKind="delimited" dfdl:terminator="%NL;">
```

Comment:
This test is not able to distinguish "." from "/", which is a key thing for tests to cover. Nest this uuid element inside another root element and then this would have more coverage, or add a second such test which would test exactly the issue of whether "." reflects the right infoset location in the schematron rules.

Reply:
Updated.  Is that change enough to cover that the rule is properly scoped to the context?

Working on improving my daffodil schema understanding at the same time as adding more complex tests and working with the JPEG schema...  :turtle: